### PR TITLE
Added elements_&_tags-html file

### PR DESCRIPTION
**Pull request by SkandSingh**
Added description about what are html elements and tags and their differences

### CODE REVIEW

File: docs/html/elements_&_tags-html.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+id: elements_&_tags-HTML
+title: 
+sidebar_label: Introduction of HTML Tags and Elements
+sidebar_position: 2
+tags: [html, introduction, web development, markup language, hyper text, web pages, career opportunities, personal growth, web-development, web design, web pages, websites, career opportunities, contribute to the web, stay relevant, express yourself, learn other technologies, have fun, how to use html, steps to start using html, set up your development environment, create your first html document, learn html syntax and structure, explore html elements and-attributes]
+description: In this tutorial, you will learn about HTML Elements and HTML Tags and about their differences
+---
+
+# HTML Tags and Elements
```
Comment (Ajay-Dhangar) [inline docs/html/elements_&_tags-html.md:10]: remove this line

File: docs/html/elements_&_tags-html.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+id: elements_&_tags-HTML
+title:
```
Comment (Ajay-Dhangar) [inline docs/html/elements_&_tags-html.md:3]: added value: HTML Tags and Elements

like 

```md
id: elements-and-tags-HTML
title: HTML Tags and Elements
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]

**Comment by SkandSingh** [comment]
I have made the suggested changes please check

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
change file name: `elements_&_tags-html.md` to `elements-and-tags-html.md`

**Review by Ajay-Dhangar — COMMENTED** [review]
wait I will resolve it

**Review by Ajay-Dhangar — APPROVED** [review]
All Right and Done

**Comment by SkandSingh** [comment]
Thank you so much for you help!! could you please explain me what was the issue so that I can take care of that from next time.

**Pull request merged**